parquet-tools/src/main/java/org/apache/parquet/tools/Main.java

```diff
@@ -226,7 +226,12 @@ public void close() throws IOException {}
       die("Invalid arguments: " + ex.getMessage(), true, name, command);
     } catch (Throwable th) {
       if (debug) th.printStackTrace(Main.err);
-      die(th, false, name, command);
```

Comment:
I think it is better to just modify `die()` to print `Throwable` as exception class + message instead of doing it specifically in `Main`.